pkg/proc/proc_test.go

```diff
@@ -154,6 +154,9 @@ func TestExit(t *testing.T) {
 }
 
 func TestExitAfterContinue(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
```

Comment:
Why is this test skipped? It's only checking that we get an exit notification from the process, this shouldn't be dependent on CPU arch.

Reply:
This test passed on my device, but i still skiped it. Because there is an bug in origin `Detach()` function.  It need to investigate later for full information. mentioned in #1770.

Reply:
This is the only thing that concerns me, the rest I understand waiting for follow up patches, but a bug in Detach makes me nervous to merge this right away.

Do you have more details on what the bug is? Aside from this issue I'm alright with merging this and fixing the rest in follow up PRs. @hengwu0 will you be around and have time to continue this work in follow up patches?

Reply:
> Do you have more details on what the bug is?

It's amd64's bug, and it had not appear in arm64(or unhited when i test on arm64), traced in #1771

> will you be around and have time to continue this work in follow up patches?

Yes, i will. The stacktrace is finished, which will be the next BP(big PR).